specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-04-01-preview/examples/ApiManagementCreateSchema2.json

```diff
@@ -0,0 +1,123 @@
+{
+  "parameters": {
+    "serviceName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2021-04-01-preview",
+    "subscriptionId": "subid",
+    "schemaId": "schema1",
+    "parameters": {
+      "properties": {
+        "schemaType": "json",
+        "document": {
```

Comment:
This seems to be a swagger document. Could you add some simple json schema instead?